modules/apps/object/object-service/src/main/java/com/liferay/object/internal/search/spi/model/index/contributor/ObjectFieldModelDocumentContributor.java

```diff
@@ -38,14 +45,40 @@ public void contribute(Document document, ObjectField objectField) {
 		document.addKeyword(
 			"objectDefinitionId", objectField.getObjectDefinitionId());
 		document.addKeyword("state", objectField.isState());
+		document.addKeyword("unique", _isUnique(objectField));
 
 		document.remove(Field.USER_NAME);
 	}
 
 	@Reference
 	protected ClassNameLocalService classNameLocalService;
 
+	private boolean _isUnique(ObjectField objectField) {
```

Comment:
You can leave it if you want, this is for future PRs, but this commit is not related with buildREST because this code it is not auto generated. This should be in other commit